package.json

```diff
@@ -20,48 +20,48 @@
     }
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
+    "@babel/core": "^7.6.4",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
     "babel-loader": "^8.0.6",
     "clean-webpack-plugin": "^3.0.0",
     "concurrently": "^4.1.2",
     "copyfiles": "^2.1.1",
     "gh-pages": "^2.1.1",
-    "husky": "^3.0.4",
+    "husky": "^3.0.9",
     "i18n-webpack-plugin": "^1.0.0",
-    "nodemon": "^1.19.1",
+    "nodemon": "^1.19.4",
     "prettier": "^1.18.2",
-    "pretty-quick": "^1.11.1",
+    "pretty-quick": "^2.0.0",
     "react-portal": "^4.2.0",
     "rimraf": "^3.0.0",
-    "webpack": "^4.39.3",
-    "webpack-bundle-analyzer": "^3.4.1",
-    "webpack-cli": "^3.3.7",
-    "webpack-dev-server": "^3.8.0"
+    "webpack": "^4.41.2",
+    "webpack-bundle-analyzer": "^3.6.0",
+    "webpack-cli": "^3.3.9",
+    "webpack-dev-server": "^3.9.0"
   },
   "dependencies": {
     "busboy": "^0.3.1",
     "classnames": "^2.2.6",
     "ejs": "^2.7.1",
     "email-validator": "^2.0.4",
     "express": "^4.17.1",
-    "firebase": "^6.4.2",
+    "firebase": "^7.2.2",
```

Comment:
See https://firebase.google.com/support/release-notes/js#version_700_-_september_26_2019 - the breaking change does not appear to impact us.